src/test/kotlin/uk/gov/dluhc/printapi/config/LocalStackContainerConfiguration.kt

```diff
@@ -214,4 +216,17 @@ class LocalStackContainerConfiguration {
         val mappedPort = getMappedPort(DEFAULT_PORT)
         return URI("http://$ipAddress:$mappedPort")
     }
+
+    protected fun createSchedulerLockTable(dynamoDbClient: DynamoDbClient, tableName: String) {
```

Comment:
Minor - `protected`? Could it be `private` ?

Reply:
Yeah it can be